nifi/nifi-nar-bundles/nifi-kafka-bundle/nifi-kafka-processors/src/main/java/org/apache/nifi/processors/kafka/PutKafka.java

```diff
@@ -136,6 +136,68 @@
             .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
             .expressionLanguageSupported(false)
             .build();
+    public static final PropertyDescriptor PRODUCER_TYPE = new PropertyDescriptor.Builder()
+            .name("Producer Type")
+            .description("This parameter specifies whether the messages are sent asynchronously in a background thread."
```

Comment:
Would put the descriptions of these two values in the Allowable Values themselves, rather than describing the possible options in the Description of the property